src/stitcher/image_loader.rs

```diff
@@ -17,16 +17,16 @@ use thiserror::Error;
 /// Errors related to loading images.
 pub enum ImageLoaderError {
```

Comment:
Removed usage of `PathBuf` here because I figured it's usually unnecessary to *also* provide the path. If necessary in the future, we can add an `Option<PathBuf>` instead.

Removing `PathBuf` in the error also allows for a `From<io::Error>` implementation.